### PR TITLE
bump(source-to-image):

### PR DESCRIPTION
**Pull request by adambkaplan**
Fix issue where s2i images with an assemble-user label can bypass the root user check in build containers.
Bug [1582976](https://bugzilla.redhat.com/show_bug.cgi?id=1582976)

/assign @deads2k 
/cc @bparees @liggitt

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -962,7 +962,7 @@ imports:
   - pkg/controller/servingcert
   - pkg/controller/servingcert/cryptoextensions
 - name: github.com/openshift/source-to-image
-  version: 27f0729d81fe0641486257252189b8dca856ea28
+  version: cfe7e9f503d38ceca35cc7f04fef5fdd822e8c5a
```
Comment (deads2k) [inline glide.lock]: This was really the only thing that moved?  You didn't cherry-pick your folder, right?

File: glide.yaml [hunk 2]
```diff
@@ -47,6 +47,8 @@ import:
   version: master
 - package: github.com/openshift/service-serving-cert-signer
   version: release-3.10
+- package: github.com/openshift/source-to-image
+  version: release-3.10
```
Comment (deads2k) [inline glide.yaml]: Why doesn't this float with master?  Why did this change? We already had this dep, didn't we?

File: glide.lock (hunk 1 above)
Comment (adambkaplan) [reply glide.lock]: no - I manually backed out several updates that were unrelated to s2i. This PR is to merge in a specific security-related fix.

File: glide.yaml (hunk 2 above)
Comment (adambkaplan) [reply glide.yaml]: We're pinning until the release-3.10 branch is cut. s2i has a new feature in it that we don't want added to origin until 3.11.

File: glide.lock (hunk 1 above)
Comment (deads2k) [reply glide.lock]: > no - I manually backed out several updates that were unrelated to s2i. This PR is to merge in a specific security-related fix.

That's a timebomb for everyone following you and we have to hope you got the cherrypicking right.  Please include everything.

File: glide.yaml (hunk 2 above)
Comment (deads2k) [reply glide.yaml]: > We're pinning until the release-3.10 branch is cut. s2i has a new feature in it that we don't want added to origin until 3.11.

ok.

File: glide.lock (hunk 1 above)
Comment (adambkaplan) [reply glide.lock]: I'll close and make a new PR as bump(*) then.

**Pull request closed without merging**